reference/constraints/Range.rst

```diff
@@ -346,15 +346,17 @@ maxMessage
 **type**: ``string`` **default**: ``This value should be {{ limit }} or less.``
 
 The message that will be shown if the underlying value is more than the
-`max`_ option, and no `min`_ option has been defined (if both are defined, use
-`notInRangeMessage`_).
+`max`_ option, and **no** `min`_ option has been defined.
+
+If **both** are defined, use `notInRangeMessage`_ or use the parameter ``{{ max }}`` instead.
```

Comment:
The "or use the parameter ``{{ max }}`` instead" part does not look right to me.

Reply:
what do you suggest instead?

Reply:
Maybe I am misreading the code, but is there actually an alternative to using `notInRangeMessage`?

Reply:
yes exactly. That's what I am trying to say with this MR. It is just not documented.. 
You can still choose to use `minMessage` and `maxMessage` but with with `{{ min }}` respectively `{{ max }}` instead of `{{ limit }}` if you choose to set both parameters.

This is confusing, for sure. And maybe this could be changed (the behavior of the validator) but this is just status quo since 4.4

Reply:
I think we should not document deprecated usages.

> If **both** options are defined, use `notInRangeMessage`_ instead.

Shortening the sentence to this should IMO be enough.

Reply:
Updated the text. I didn't know that the usage was deprecated. Deprecating this feature doesn't really make sense to me as I think it's a pretty useful feature to either show one of the messages and not always both.. Currently the user has the choice, in future the user is forced to always show "min" and "max" in one sentence. Not really an improvement imho.

Reply:
If you think the deprecation should be reverted for good reasons, can you please open an issue on symfony/symfony and explain the use cases there? Thanks

In this case we should not merge this PR for now and wait the decisions to avoid back and forth here 👍🏻

@xabbuh id this is deprecated, shouldn't this be mentioned in the docs? 🧐

@cabb